src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
```

Comment:
Perhaps you could leave a new line to group your imports, similar to your other files